<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZMarkSwcSomaDialog</class>
 <widget class="QDialog" name="ZMarkSwcSomaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mark SWC Soma</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>321</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="voxelSizeXYLabel">
      <property name="text">
       <string>Voxel Size (XY)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="voxelSizeXYDoubleSpinBox">
      <property name="suffix">
       <string>μm</string>
      </property>
      <property name="decimals">
       <number>7</number>
      </property>
      <property name="minimum">
       <double>0.000001000000000</double>
      </property>
      <property name="maximum">
       <double>99999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="radiusThresholdLabel">
      <property name="text">
       <string>Soma Radius Threshold</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="radiusThresholdDoubleSpinBox">
      <property name="suffix">
       <string>μm</string>
      </property>
      <property name="decimals">
       <number>7</number>
      </property>
      <property name="maximum">
       <double>9999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>3.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="somaNodesTypeLabel">
      <property name="text">
       <string>Soma Nodes Type</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="somaNodesTypeSpinBox">
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="otherNodesTypeLabel">
      <property name="text">
       <string>Other Nodes Type</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="otherNodesTypeSpinBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span&gt;Set different node type for soma nodes and other &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span&gt;nodes. To visualize the result, change Neurons &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span&gt;render setting &amp;quot;Color Mode&amp;quot; to &amp;quot;Branch Type&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ZMarkSwcSomaDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ZMarkSwcSomaDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
